jobs/e2e_node/containerd/containerd-master/env-cgroupv2

```diff
@@ -0,0 +1,10 @@
+CONTAINERD_TEST: 'true'
```

Comment:
nit: maybe consider putting all the cgroup config under a dedicated cgroupv2 directory, i.e.

```
jobs/e2e_node/containerd/containerd-master/cgroupv2/env-cgroupv2
jobs/e2e_node/containerd/containerd-master/cgroupv2/image-config-cgroupv2.yaml
```

@dims thoughts? any particular preference here on config location?

Reply:
Thanks for the suggestion. Will move it to a dedicated cgroupv2 dir.